docs/UserGuide.md

```diff
@@ -5,44 +5,111 @@ title: User Guide
+[here](https://docs.oracle.com/en/java/javase/11/install/overview-jdk-installation.html#GUID-8677A77F-231A-40F7-98B9-1FD0B48C346A).
+
+2. Next, download the latest `gitgud.jar` from [here](https://github.com/AY2122S1-CS2103T-W13-4/tp/releases).
 
-2. Download the latest `gitgud.jar` from [here](https://github.com/AY2122S1-CS2103T-W13-4/tp/releases).
 
 3. Copy the file to the folder you want to use as the _home folder_ for your gitGud application.
 
+
 4. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds.<br>
-   ![Ui](images/ui-snapshots/startup.png)
-<br><ins>gitGud upon startup.</ins>
-5. Type the command in the command box and press `Enter` to execute it. e.g. typing **`help`** and pressing `Enter` will open the help window.<br>
-   Some example commands you can try:
+   ![Ui](images/ui-screenshots/startup.png)
+<br><center><ins>Image: gitGud upon startup.</ins></center>
+
+5. Choose and type in a command into the command input box and press `Enter` to execute it. 
+e.g. typing **`help`** and pressing `Enter` will open the help window.<br>
+   Here are some example commands you can try:
 
    * **`friend --list`** : Lists all friends.
 
-   * **`friend --add`**`Draco -n Marcus` : Adds a friend __Marcus__ with gitGud `FRIEND_ID` of __Draco__ to the application.
+   * **`friend --add Draco -n Marcus`** : Adds a friend __Marcus__ with gitGud `FRIEND_ID` of __Draco__ to the application.
 
-   * **`friend --delete`**`Draco` : Deletes friend with gitGud `FRIEND_ID` of __Draco__.
+   * **`friend --delete Draco`** : Deletes friend with gitGud `FRIEND_ID` of __Draco__.
 
    * **`exit`** : Exits the app.
-
-6. Refer to the [Features](#features) below for details of each command.
+   
+Congratulations! You have successfully set up gitGud.
+For more in-depth details of each command, refer to the [features](#features) section below.
 
 --------------------------------------------------------------------------------------------------------------------
+## gitGud User Interface 
+
+Before we jump into the commands, let us take a moment to familiarise ourselves with the user interface. 
 
+gitGud provides a gamer-themed user interface as shown in the image below with 6 main sections.
+
+![Ui-labelled](images/ui-screenshots/ui-overview.png)
+<br><center><ins>Image: User interface layout of gitGud.</ins></center>
+
+| Number | Section Name | Description |
+| -------| -------------- | ----------- | 
+| 1 | Toolbar | Currently provides functionality related to exiting and on accessing the help screen. | 
+| 2 | Command input box | This is where you can input commands to perform actions in gitGud. |
+| 3 | Command feedback box | This box displays information related to the status of the commands you have executed. |
+| 4 | Friends list | This section displays the friends list - and may be filtered to display specific friends based on the previous executed commands. |
```

Comment:
Minor nitpick, consider removing the dash if its not necessary, if not I think its fine as well.